docs/src/pages/examples/rag/chunk-markdown.mdx

```diff
@@ -1,13 +1,13 @@
 ---
 title: "Example: Semantically Chunking Markdown | RAG | Mastra Docs"
-description: Example of using Mastra to intelligently chunk markdown documents for search or retrieval purposes.
+description: Example of using Mastra to chunk markdown documents for search or retrieval purposes.
 ---
 
 import { GithubLink } from "../../../components/github-link";
 
 # Chunk Markdown
 
-Markdown is more information-dense than raw HTML, making it easier to work with for RAG pipelines. When working with markdown, you need to split it into smaller pieces while preserving headers and formatting. The `chunk` method handles Markdown-specific elements like headers, lists, and code blocks intelligently. This example shows how to chunk markdown documents for search or retrieval purposes.
+Markdown uses a compact syntax compared to HTML. The `chunk` method segments Markdown content by preserving structural elements such as headers, lists, and code blocks. This example demonstrates Markdown document chunking for search and retrieval.
```

Comment:
Please revert the changes to these page header paragraphs, the old ones are better